File: app/src/main/java/com/bunker/bunker/model/CalendarModel.java
package com.bunker.bunker.model;

import com.google.firebase.database.Exclude;
import com.google.firebase.database.IgnoreExtraProperties;

import java.util.HashMap;
import java.util.Map;

@IgnoreExtraProperties
public class CalendarModel {
  public int NoPoliza;
  public String Nombre;
  public String Beneficiario;
  public double Monto;
  // 1-31
  public int Dia;
  // 1-12
  public int Mes;
  //
  public String Telefono;
  public String Email;
  public boolean HasStar;
  public String Compania;

  // Monthly 1-12()
  public int Plan;

  public CalendarModel() {
  }

  @Exclude
  public Map<String, Object> toMap() {
    HashMap<String, Object> result = new HashMap<>();
    result.put("NoPoliza", NoPoliza);
    result.put("Nombre", Nombre);
    result.put("Beneficiario", Beneficiario);
    result.put("Monto", Monto);
    result.put("Dia", Dia);
    result.put("Mes", Mes);
    result.put("Telefono", Telefono);
    result.put("Email", Email);
    result.put("HasStar", HasStar);
    result.put("Compania", Compania);
    result.put("Plan", Plan);
    return result;
  }
}

File: app/src/main/java/com/bunker/bunker/model/CalendarModel.kt
package com.bunker.bunker.model

import com.google.firebase.database.Exclude
import com.google.firebase.database.IgnoreExtraProperties

import java.util.HashMap

@IgnoreExtraProperties
class CalendarModel {
  var NoPoliza: Int = 0
  var Nombre: String = ""
  var Beneficiario: String = ""
  var Monto: Double = 0.0
  // 1-31
  var Dia: Int = 0
  // 1-12
  var Mes: Int = 0
  //1900->
  var Year: Int = 0
  //
  var Telefono: String = ""
  var Email: String = ""
  var HasStar: Boolean = false
  var Compania: String = ""

  // Monthly 1-12()
  var Plan: Int = 0

  @Exclude
  fun toMap(): Map<String, Any> {
    val result = HashMap<String, Any>()
    result["NoPoliza"] = NoPoliza
    result["Nombre"] = Nombre
    result["Beneficiario"] = Beneficiario
    result["Monto"] = Monto
    result["Dia"] = Dia
    result["Mes"] = Mes
    result["Year"] = Year
    result["Telefono"] = Telefono
    result["Email"] = Email
    result["HasStar"] = HasStar
    result["Compania"] = Compania
    result["Plan"] = Plan
    return result
  }
}